react/I18n/format.jsx

```diff
@@ -3,26 +3,31 @@ import { DEFAULT_LANG } from '.'
 import formatDistanceToNow from 'date-fns/formatDistanceToNow'
 
 const locales = {}
-let lang = DEFAULT_LANG
+let lang = DEFAULT_LANG === 'en' ? 'en-US' : DEFAULT_LANG
 
 const getWarningMessage = lang =>
   `The "${lang}" locale isn't supported by date-fns. or has not been included in the build. Check if you have configured a ContextReplacementPlugin that is too restrictive.`
 
 export const provideDateFnsLocale = (userLang, defaultLang = DEFAULT_LANG) => {
-  lang = userLang
+  const resolvedDefaultLang = defaultLang === 'en' ? 'en-US' : defaultLang
+  const resolvedUserLang = userLang === 'en' ? 'en-US' : userLang
```

Comment:
This should also be done for `fr-FR`, right? And also `es-ES` since these are the 3 supported langages

Reply:
French and Spanish localizations have not been moved. They are still in `fr` and `es` directory and `fr-FR` and `es-ES` do not exist, so we should not change anything for them.